runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -270,26 +270,26 @@ MM_CopyForwardScheme::initialize(MM_EnvironmentVLHGC *env)
 
 	/* Create and initialize the owned region lists to maintain resource for survivor area heap acquisition */
 	_reservedRegionList = (MM_ReservedRegionListHeader *)env->getForge()->allocate(sizeof(MM_ReservedRegionListHeader) * _compactGroupMaxCount, MM_AllocationCategory::FIXED, J9_GET_CALLSITE());
-	if(NULL == _reservedRegionList) {
+	if (NULL == _reservedRegionList) {
 		return false;
 	}
 	
 	memset((void *)_reservedRegionList, 0, sizeof(MM_ReservedRegionListHeader) * _compactGroupMaxCount);
-	for(UDATA index = 0; index < _compactGroupMaxCount; index++) {
+	for (UDATA index = 0; index < _compactGroupMaxCount; index++) {
```

Comment:
well, now that you fixed spacing in this huge file (which I did not expect), you could go and fix UDATA, too :)

Reply:
I afraid such update might hide original change, Spaces can be filtered out from view. I suggest to do update of types in separate PR